framework/src/main/java/org/tron/core/net/messagehandler/BlockMsgHandler.java

```diff
@@ -76,6 +76,7 @@ public void processMessage(PeerConnection peer, TronMessage msg) throws P2pExcep
     if (!fastForward && !peer.isRelayPeer()) {
       check(peer, blockMessage);
     }
+    peer.setLastActiveTime(System.currentTimeMillis());
```

Comment:
It is recommended to implement it in the `P2pEventHandlerImpl.processMessage` method, otherwise there are too many places to add.